### PR TITLE
FE: Replace 'FeatherTab' to toggle between user mangement and integrations component with the one we're using in Discovery component.

### PR DESCRIPTION
**Pull request by atif1019**
## Description
Initially, we used 'FeatherTab' to toggle between user management and integration components. But according to figma we need to create a custom component. 

Split page in two sections. 

Left section will contains a card that has 2 items User Management and Integrations.

Upon selection of item on left section right section will change and display the respective component.

See Discovery compenont for help.

## Jira link(s)
- https://opennms.atlassian.net/browse/LOK-2671

### CODE REVIEW

**Review by Shahbaz-dataq — APPROVED** [review]
LGTM

File: ui/src/components/Settings/ToggleCard.vue [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import { defineProps, defineEmits } from 'vue'
+
+defineProps<{
+  title: string;
+  isActive?: boolean;
+}>()
+
+const emit = defineEmits(['toggle'])
+
+function handleToggle() {
```
Comment (synqotik) [inline ui/src/components/Settings/ToggleCard.vue:17]: Use this syntax: `const handleToggle = () => {`

File: ui/src/containers/Settings.vue [hunk 2]
```diff
@@ -8,26 +8,48 @@
...
   </div>
 </template>
 
 <script lang="ts" setup>
-import { FeatherTab, FeatherTabContainer, FeatherTabPanel } from '@featherds/tabs'
+import { ref } from 'vue'
+
+const activeToggle = ref('userManagement')
+
+function setActiveToggle(toggle: string) {
```
Comment (synqotik) [inline ui/src/containers/Settings.vue:40]: Use this syntax: `const setActiveToggle = (toggle: string) => {`

File: ui/src/containers/Settings.vue [hunk 3]
```diff
@@ -8,26 +8,48 @@
...
+  activeToggle.value = toggle
+}
 </script>
 
 <style lang="scss" scoped>
 @use '@featherds/styles/themes/variables';
 @use '@/styles/vars';
-
+.custom-tabs-container{
```
Comment (synqotik) [inline ui/src/containers/Settings.vue:48]: Space before `{`

File: ui/src/containers/Settings.vue [hunk 4]
```diff
@@ -8,26 +8,48 @@
...
+  display: flex;
+}
+.management-content{
```
Comment (synqotik) [inline ui/src/containers/Settings.vue:51]: Space before `{`

File: ui/src/components/Settings/ToggleCard.vue [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+.toggle-card {
+  cursor: pointer;
+  padding: 10px;
+  border: 1px solid var(variables.$border-on-surface);
+  border-radius: vars.$border-radius-surface;
+  background-color: var(variables.$surface);
+  width: 18vw;
+}
+
+.toggle-card.active {
```
Comment (synqotik) [inline ui/src/components/Settings/ToggleCard.vue:35]: Should put this inside `.toggle-card` block, either as `.active` or `&.active`

**Review by synqotik — CHANGES_REQUESTED** [review]
Just a few small syntax fixes, ping me when complete.

**Review by synqotik — APPROVED** [review]
LGTM

**Pull request merged**